pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -1454,6 +1504,9 @@ type TimeConfig struct {
 	//     Specifies the timeout when the node time is considered to be in sync unlocking the boot sequence.
 	//     NTP sync will be still running in the background.
 	//     Defaults to "infinity" (waiting forever for time sync)
+	//   schema:
+	//     type: string
+	//     pattern: ^[-+]?(((\d+(\.\d*)?|\d*(\.\d+)+)(ns|us|µs|ms|s|m|h))|0)+$
```

Comment:
Test of this pattern: https://regex101.com/r/xNTblv/2

Reply:
that's a lot of matches :cool:

Reply:
yes, took them from Go's internal parse tests. It matches all except the ones fail due to overflow.